### ISSUE TITLE
Clairfy Fig 3.1 in software user manual v2.0

### ISSUE THREAD

**Issue by versatran01** [ouster-lidar/ouster_example#201]
I have problem understanding the equations that compute xyz in page 12 of software user manual 2.0
![image](https://user-images.githubusercontent.com/1883423/103319339-857d4680-49ff-11eb-9f7e-e81f64b53847.png)

![image](https://user-images.githubusercontent.com/1883423/103319356-9332cc00-49ff-11eb-8286-f6212fecbdd4.png)

![image](https://user-images.githubusercontent.com/1883423/103319363-9a59da00-49ff-11eb-86c1-974bacf5194f.png)

So my question is, what is `r`? Is it the distance to the beam origin (the rotating block)? Found no clear definition in the user manual.
If so, then why is `n` involved in the computation of `z`?
Basically confused by the `(r-n)` term.

**Comment by dllu** [comment]
The definition of `r` is literally right above the equations that you screenshotted.

![image](https://user-images.githubusercontent.com/14482624/103499315-77af3300-4dfc-11eb-93c5-4f387cab61e1.png)

**Comment by versatran01** [comment]
I saw that but is it range to the beam origin or to the center of the sensor?

**Comment by versatran01** [comment]
@dgarbuzov any comments?

**Comment by dgarbuzov** [comment]
The reported range `r` is the full distance to the center of the sensor, including `n` (the distance between the lidar origin and beam origin). In the equations for `x` and `y`, `r - n` is the measured range from the beam origin, so the first term gives you the coordinate in the frame of the beam origin and and the second term accounts for the offset to the lidar origin.

Hope that helps. I agree the label for `r` in the diagram is misleading -- sorry about that!
